### PR TITLE
Make docs/contribute/_index.md follow v1.19 of the original text

### LINKED ISSUE #26903: ja: Make docs/contribute/_index.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/contribute/_index.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/contribute/_index.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/contribute/_index.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/contribute/_index.md

diff between translated and v1.19
https://gist.github.com/7a7f4f26df56b52b4cd65ece764700a5

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by dragoneena12**
closes #26903
update `docs/contribute/_index.md` to follow v1.19

### CODE REVIEW

File: content/ja/docs/contribute/_index.md [hunk 1]
```diff
@@ -1,55 +1,76 @@
...
-    - [chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.com/blog/1506-closing-issues-via-pull-requests )
-    - [davidwalsh.name/squash-commits-git ](https://davidwalsh.name/squash-commits-git )
+## はじめての貢献
+- 貢献のための複数の方法について学ぶために[貢献の概要](/docs/contribute/new-content/overview/)を読んでください。
+- 良い開始地点を探すために[`kubernetes/website` issueリスト](https://github.com/kubernetes/website/issues/)を確認してください。
+- 既存のドキュメントに対して[GitHubを使ってプルリクエストをオープン](/docs/contribute/new-content/open-a-pr/#changes-using-github)し、GitHubへのissueの登録について学んでください。
+- 正確さと言語の校正のため、他のKubernetesコミュニティメンバーから[プルリクエストのレビュー](/docs/contribute/review/reviewing-prs/)を受けてください。
+- 見識のあるコメントを残せるようにするため、Kubernetesの[コンテンツ](/docs/contribute/style/content-guide/)と[スタイルガイド](/docs/contribute/style/style-guide/)を読んでください。
+- [ページコンテンツの種類](/docs/contribute/style/page-content-types/)と[Hugoショートカット](/docs/contribute/style/hugo-shortcodes/)について勉強してください。
```
Comment (atoato88) [inline content/ja/docs/contribute/_index.md:53]: Original word is `shortcode` not `shortcut`, is the translation of `ショートカット` on purpose?

**Review by atoato88 — COMMENTED** [review]
Almost LGTM, just one comment.

File: content/ja/docs/contribute/_index.md (hunk 1 above)
Comment (dragoneena12) [reply content/ja/docs/contribute/_index.md]: It's my mistake. Thank you!

**Comment by atoato88** [comment]
/lgtm
Thank you for quick fixing!

**Pull request merged**